### PR TITLE
Added QChem to Zeus Scripts

### PR DESCRIPTION
**Pull request by calvinp0**
Added QChem to Zeus submit script

### CODE REVIEW

File: Servers/Zeus/ARC/.arc/submit.py [hunk 1]
```diff
@@ -34,7 +34,11 @@
 
 cp "$PBS_O_WORKDIR/input.gjf" $GAUSS_SCRDIR
 
-cp "$PBS_O_WORKDIR/check.chk" $GAUSS_SCRDIR
+if [ ! -f check.* ]; then
+	:
```
Comment (Lilachn91) [inline Servers/Zeus/ARC/.arc/submit.py]: Shouldn't it be the other way around - if the check file doesn't exist in the scratch directory, then copy it there?

File: Servers/Zeus/ARC/.arc/submit.py [hunk 2]
```diff
@@ -90,6 +94,56 @@
...
+# Create a file to measure the time of execution
+touch initial_time
+
+# Change directory to the VM storage
+cd $SCRATCH
+
+# Run QChem
+qchem -pbs -nt {cpus} input.in output.out
+
+# Copy all the files back to the current directory
```
Comment (Lilachn91) [inline Servers/Zeus/ARC/.arc/submit.py:131]: Is it OK to copy all the files? Qchem doesn't create temporary files that we don't want to keep?

**Review by Lilachn91 — COMMENTED** [review]
Good job :)
See two questions I had.

File: Servers/Zeus/ARC/.arc/submit.py (hunk 2 above)
Comment (calvinp0) [reply Servers/Zeus/ARC/.arc/submit.py:131]: As far as I am aware, it doesn't create temp. files. I have done a few runs on my Zeus drive and haven't had anything extra come through.

File: Servers/Zeus/ARC/.arc/submit.py (hunk 1 above)
Comment (calvinp0) [reply Servers/Zeus/ARC/.arc/submit.py]: Do you mean by making placing the if condition here:

https://github.com/DanaResearchGroup/DRGScripts/blob/25ea111fee53f9cf78a2ba191973c956a8fb5443/Servers/Zeus/ARC/.arc/submit.py#L35

File: Servers/Zeus/ARC/.arc/submit.py (hunk 2 above)
Comment (Lilachn91) [reply Servers/Zeus/ARC/.arc/submit.py:131]: OK. Cool, thanks :)

**Review by Lilachn91 — APPROVED** [review]
Looks good :) Thanks!

**Pull request merged**